packages/core/src/scatter-chart.ts

```diff
@@ -74,8 +74,12 @@ export class ScatterChart extends BaseAxisChart {
 		// Or the chart is only a scatter chart
 		// And not a line chart for instance
 		const circleShouldBeFilled = radius < Configuration.lines.points.minNonFilledRadius || this.constructor === ScatterChart;
-
-		return circleShouldBeFilled ? this.getStrokeColor(d.datasetLabel, d.label, d.value) : "white";
+		if (circleShouldBeFilled) {
+			return this.getStrokeColor(d.datasetLabel, d.label, d.value);
+		}
+		// return the chart-grid-backdrop color or default to white
+		const circleFillColor = this.innerWrap.selectAll("rect.chart-grid-backdrop").style("fill");
```

Comment:
This seems a bit hacky. What if in the future the circles render before the grid is rendered?

Can't the fill be applied through CSS (same as the backdrop), and if we need to override with white, we can still do that here in JS?

Reply:
yes, done. 👍 
returning null instead since setting an attribute to null will discard setting that value.  Set the colors in the theme scss files instead which will take priority